libs/librrgraph/src/base/rr_graph_type.h

```diff
@@ -0,0 +1,36 @@
+#ifndef GRAPH_TYPE_H
+#define GRAPH_TYPE_H
+
+#include <vector>
+#include "physical_types.h"
```

Comment:
It seems to me that you do not need any data structure or data types from the ``physical_types.h`` in this header file (just consider the data types in the header file, not the source file).
If it is true, I suggest to remove this header inclusion.

Reply:
`t_unified_to_parallel_seg_index` requires `e_parallel_axis` from `physical_types.h`

Reply:
o.k. Not a problem then.